content/en/blog/2021/proxyless-grpc/index.md

```diff
@@ -0,0 +1,400 @@
+---
+title: "gRPC Proxyless Service Mesh"
+description: Introduction to Istio support for gRPC's proxyless service mesh features.
+publishdate: 2021-09-02
+attribution: "Steven Landow (Google)"
+---
+
+Istio dynamically configures its Envoy sidecar proxies using a set of discovery APIs, collectively known as the
+[xDS APIs](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/operations/dynamic_configuration).
+These APIs aim to become a [universal data-plane API](https://blog.envoyproxy.io/the-universal-data-plane-api-d15cec7a?gi=64aa2eea0283).
+The gRPC project has started to implement the xDS APIs, which means you can manage gRPC workloads without having to deploy
```

Comment:
gRPC already has significant support for xDS APIs. Please replace "has started to implement" with "already has significant support for" and link to https://github.com/grpc/proposal/search?q=xds

Reply:
sure thing!